draft-ietf-quic-transport.md

```diff
@@ -3433,24 +3434,32 @@ non-ack-eliciting packets need to be acknowledged, an endpoint MAY wait until an
 ack-eliciting packet has been received to bundle an ACK frame with outgoing
 frames.
 
-The algorithms in {{QUIC-RECOVERY}} are resilient to receivers that do not
-follow guidance offered above. However, an implementor should only deviate from
-these requirements after careful consideration of the performance implications
-of doing so.
 
-Packets containing only ACK frames are not congestion controlled, so there are
-limits on how frequently they can be sent.  An endpoint MUST NOT send more than
-one ACK-frame-only packet in response to receiving an ack-eliciting packet.  An
-endpoint MUST NOT send a non-ack-eliciting packet in response to a
-non-ack-eliciting packet, even if there are packet gaps which precede the
-received packet. Limiting ACK frames avoids an infinite feedback loop of
-acknowledgements, which could prevent the connection from ever becoming idle.
-However, the endpoint acknowledges non-ACK-eliciting packets when it sends an
-ACK frame.
+### Acknowledgement Frequency
+
+A receiver determines how frequently to send acknowledgements in response to
+ack-eliciting packets. This determination involves a tradeoff.
+
+Endpoints rely on timely acknowledgment to detect loss; see Section 5 of
+{{QUIC-RECOVERY}}. Window-based congestion controllers, such as the one in
+Section 6 of {{QUIC-RECOVERY}}, rely on acknowledgments to manage their sending
+rate. In both cases, delaying acknowledgment can adversely affect performance.
+
+On the other hand, reducing the frequency of packets that carrying only
+acknowledgements reduces packet processing cost at both endpoints. It can also
```

Comment:
Q: Doesn't it reduce packet serialization and sending costs at the receiver and packet processing costs at the sender?  In practice, the cost of sending ACK-only packets is commonly worse, so it seems worth mentioning.

Reply:
That is a detail, which will need further substantiation. The point here is to note _some_ benefits, to point out that there are some. The specifics will depend on your particular platform.

Reply:
Now that I read this again, I'm actually not sure what you mean.  Do you mean packet processing cost as in the cost of receiving a large number of data packets, so fewer ACK-only packets reduces that cost?

I was thinking you were saying it reduces the number of ACKs that have to be received and processed.

Either way, I have a small suggestion.
```suggestion
acknowledgements reduces packet transmission and processing cost at both endpoints. It can also
```